knexfile.js

```diff
@@ -1,26 +1,35 @@
+// Update with your config settings.
+
 module.exports = {
 
   development: {
     client: 'sqlite3',
     connection: {
       filename: './dev.sqlite3'
-    },
-    useNullAsDefault: true
```

Comment:
It looks like you've re-initiated knex which has overwritten some of Rich's wonderful changes! Removing `useNullAsDefault` can cause some seeds to fail.

Reply:
I'd recommend restoring the original `knexfile.js` from development.